modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -35,5 +35,41 @@
 		],
 		"newReference": "HtmlParser _htmlParser",
 		"to": "_htmlParser.extractText"
+	},
+	{
+		"from": "CommerceOrderConstants.PAYMENT_STATUS_AUTHORIZED",
+		"issueKey": "LPS-189358",
+		"newImports": [
+			"com.liferay.commerce.constants.CommerceOrderPaymentConstants"
+		],
+		"to": "CommerceOrderPaymentConstants.STATUS_AUTHORIZED",
+		"validExtensions": [
+			"java",
+			"jsp"
+		]
+	},
+	{
+		"from": "CommerceOrderConstants.PAYMENT_STATUS_PAID",
+		"issueKey": "LPS-189358",
+		"newImports": [
+			"com.liferay.commerce.constants.CommerceOrderPaymentConstants"
+		],
+		"to": "CommerceOrderPaymentConstants.STATUS_COMPLETED",
+		"validExtensions": [
+			"java",
+			"jsp"
+		]
+	},
+	{
+		"from": "CommerceOrderConstants.PAYMENT_STATUS_PENDING",
+		"issueKey": "LPS-189358",
+		"newImports": [
+			"com.liferay.commerce.constants.CommerceOrderPaymentConstants"
+		],
+		"to": "CommerceOrderPaymentConstants.STATUS_PENDING",
+		"validExtensions": [
+			"java",
+			"jsp"
+		]
```

Comment:
@nicolas-sss, there is a chance of Brian ask us to allow the key `from` to support more than one value at once, like an array, but let's give it a try. :wink:

Reply:
I imagined this, but in order to follow the concept of each pattern being a `jsonObject`, it has to be separate. In order to follow the concept, I left it this way just to see if that's exactly what B-chan wants.